repos/corefx.proj

```diff
@@ -18,16 +18,20 @@
     <OverrideTargetRid Condition="'$(TargetOS)' == 'Windows_NT'">win-x64</OverrideTargetRid>
 
     <BuildArguments>$(FlagParameterPrefix)configuration $(Configuration)</BuildArguments>
-    <BuildArguments>$(BuildArguments) $(FlagParameterPrefix)restore $(FlagParameterPrefix)build $(FlagParameterPrefix)pack</BuildArguments>
-    <BuildArguments>$(BuildArguments) /p:SkipTests=true</BuildArguments>
+    <BuildArguments Condition="'$(RunTests)' != 'true' AND '$(PrepForTests)' != 'true'">$(BuildArguments) $(FlagParameterPrefix)restore $(FlagParameterPrefix)build $(FlagParameterPrefix)pack</BuildArguments>
+    <BuildArguments Condition="'$(RunTests)' != 'true'">$(BuildArguments) /p:SkipTests=true</BuildArguments>
+    <BuildArguments Condition="'$(PrepForTests)' == 'true'">$(BuildArguments) $(FlagParameterPrefix)build /p:IncludeTestUtils=true</BuildArguments>
+    <BuildArguments Condition="'$(RunTests)' == 'true'">$(BuildArguments) -buildtests -test</BuildArguments>
+    <BuildArguments Condition="'$(RunTests)' == 'true'">$(BuildArguments) /p:TestRspFile=$(TestExclusionsDir)corefx/linux.docker.rsp</BuildArguments>
```

Comment:
Not sure what happened, but I'm missing a commit here that conditioned this for Linux on Docker only.